backend/tests/v5/e2e/routes/teamspaces/projects/containers/containers.test.js

```diff
@@ -43,6 +44,7 @@ const models = [
 		_id: ServiceHelper.generateUUIDString(),
 		name: ServiceHelper.generateRandomString(),
 		isFavourite: true,
+		permissions: [{ user: users.limitedPermissions, permission: 'viewer' }, { user: users.limitedPermissions, permission: 'commentator' }],
```

Comment:
It's "commenter"

Reply:
also doesn't make sense for a user to be both viewer and commenter at the same time. Make 2 users